MunchkinLevelCounter/UI/Charts/ChartsPresenter.swift

```diff
@@ -26,28 +27,29 @@ class ChartsPresenter: Presenter {
     
     func loadChartData(type: ScoreType) {
         loggingPrint("Loading chart data")
-        disposable = dataManager.getLineData(type: type.rawValue)
+        dataManager.getLineData(type: type.rawValue)
             .subscribeOn(ConcurrentDispatchQueueScheduler(qos: DispatchQoS.background))
             .observeOn(MainScheduler.instance)
             .subscribe(onNext: { lineChartData in
                     self.chartsView?.showPlayersCharts(chartData: lineChartData)
                 }
             )
+            .disposed(by: disposeBag)
     }
     
     func loadPlayers(sortType: ScoreType) {
-        disposable = dataManager.getPlayers(sortType: sortType)
+        dataManager.getPlayers(sortType: sortType)
             .subscribeOn(ConcurrentDispatchQueueScheduler(qos: DispatchQoS.background))
             .observeOn(MainScheduler.instance)
             .subscribe(onNext: { players in
                     self.chartsView?.showPlayersList(players: players)
                 }
             )
+            .disposed(by: disposeBag)
     }
     
     func detachView() {
         chartsView = nil
-        disposable?.dispose()
```

Comment:
If I'm not mistaken, if you want to destroy DisposableBag, then you have to set the reference to nil.
References:
http://rx-marin.com/post/rxswift-timer-sequence-manual-dispose-bag/
https://stackoverflow.com/questions/38969328/manually-disposing-a-disposebag-in-rxswift